edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.CloudProxy/CloudConnectionProvider.cs

```diff
@@ -15,6 +15,9 @@ namespace Microsoft.Azure.Devices.Edge.Hub.CloudProxy
 
     public class CloudConnectionProvider : ICloudConnectionProvider
     {
+        // Minimum value allowed by the SDK for Connection Idle timeout for AMQP Multiplexed connections.
+        static readonly TimeSpan MinAmqpConnectionMuxIdleTimeout = TimeSpan.FromSeconds(5);
```

Comment:
Should this be configurable?

Reply:
Emm, the idea was to keep this to the minimum value allowed, so that the connections are cleaned up as fast as possible. I checked with Rajeev as well, and he agreed this should be the default for EdgeHub to handle the offline/online reconnect scenario.

Reply:
Ok cool